alacritty.spec.rpkg

```diff
@@ -0,0 +1,56 @@
+Name:          {{{ git_name }}}
+Version:       {{{ git_version }}}
+Release:       1%{?dist}
+Summary:       A cross-platform, GPU enhanced terminal emulator
+License:       ASL 2.0
```

Comment:
I'm not sure what the ASL 2.0 license is, but I don't think it's the Apache License which Alacritty is licensed under.

Reply:
"ASL" appears to stand for "Apache Software License" -- see https://wiki.debian.org/DFSGLicenses#The_Apache_Software_License_.28ASL.29 (one of the few references to that nomenclature).

Reply:
Huh, interesting. I usually only see it shortened to Apache-2.0 and that seems to be what most websites refer to it as.

Though if Debian says that's good, this is fine by me!

Reply:
SPDX agrees with you (and "most websites"), so it may be better to use that anyways. See the "Short Identifier" at https://spdx.org/licenses/Apache-2.0.html

Reply:
Yeah I'd say if @pschyska has picked this for a specific reason, it's fine. But if this isn't a Fedora requirement the Apache-2.0 name is probably better.

Reply:
I think they prefer ASL, see https://fedoraproject.org/wiki/Licensing:Main. That files is linked from COPR because you are only allowed to build software with licenses mentioned in that wiki there.

Reply:
Just double checked that a quick google for "ASL 2.0" didn't give me some revision of the **A**merican **S**ign **L**anguage. :rofl: